Mail/Helpers/RichTextEditor.swift

```diff
@@ -96,11 +96,29 @@ struct RichTextEditor: UIViewRepresentable {
             }
         }
 
+        @MainActor
         func editorContentChanged(_ editor: SQTextEditorView, content: String) {
-            var parentBody = parent.body.trimmingCharacters(in: .whitespacesAndNewlines)
-            parentBody = parentBody.replacingOccurrences(of: "\r", with: "")
-            if parentBody != content {
-                parent.body = content
+            let parentBody = parent.body
+            Task.detached {
```

Comment:
IMO we need the detached here, the larger the content edited / pasted the more we need it.